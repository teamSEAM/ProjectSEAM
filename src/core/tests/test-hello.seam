function main():
	print("Hello world!")
	print("We can print two lines too.")
	print("Edwards, do we get full credit now?")
