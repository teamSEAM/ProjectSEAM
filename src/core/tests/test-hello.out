Hello world!
We can print two lines too.
Edwards, do we get full credit now?
